addons/hr/models/res_users.py

```diff
@@ -311,3 +311,22 @@ def action_create_employee(self):
             company_id=self.env.company.id,
             **self.env['hr.employee']._sync_user(self)
         ))
+
+    def action_open_employees(self):
+        self.ensure_one()
+        employees = self.employee_ids
+        if len(employees) > 1:
+            return {
+                'name': _('Employee'),
```

Comment:
To stay consistent with the other action :-)

```suggestion
                'name': _('Related Employees'),
```